packages/flutter/lib/src/cupertino/icons.dart

```diff
@@ -151,109 +151,109 @@ class CupertinoIcons {
   static const IconData home = IconData(0xf447, fontFamily: iconFont, fontPackage: iconFontPackage);
 
-  /// A right-facing shopping cart outline.
+  /// <i class='cupertino-icons md-36'>cart</i> &#x2014; cupertino icon for a right-facing shopping cart outline.
   /// This is the same icon as [cart] in cupertino_icons 1.0.0+.
   static const IconData shopping_cart = IconData(0xf3f7, fontFamily: iconFont, fontPackage: iconFontPackage);
 
-  /// Three solid dots.
+  /// <i class='cupertino-icons md-36'>ellipsis</i> &#x2014; cupertino icon for three solid dots.
   static const IconData ellipsis = IconData(0xf46a, fontFamily: iconFont, fontPackage: iconFontPackage);
 
-  /// A phone handset outline.
+  /// <i class='cupertino-icons md-36'>phone</i> &#x2014; cupertino icon for a phone handset outline.
   static const IconData phone = IconData(0xf4b8, fontFamily: iconFont, fontPackage: iconFontPackage);
 
-  /// A phone handset.
+  /// <i class='cupertino-icons md-36'>phone_fill</i> &#x2014; cupertino icon for a phone handset.
   /// This is the same icon as [phone_fill] in cupertino_icons 1.0.0+.
   static const IconData phone_solid = IconData(0xf4b9, fontFamily: iconFont, fontPackage: iconFontPackage);
 
-  /// A solid down arrow.
+  /// <i class='cupertino-icons md-36'>arrow_down</i> &#x2014; cupertino icon for a solid down arrow.
   /// This is the same icon as [arrow_down] in cupertino_icons 1.0.0+.
   static const IconData down_arrow = IconData(0xf35d, fontFamily: iconFont, fontPackage: iconFontPackage);
 
-  /// A solid up arrow.
+  /// <i class='cupertino-icons md-36'>arrow_up</i> &#x2014; cupertino icon for a solid up arrow.
   /// This is the same icon as [arrow_up] in cupertino_icons 1.0.0+.
   static const IconData up_arrow = IconData(0xf366, fontFamily: iconFont, fontPackage: iconFontPackage);
 
-  /// A charging battery.
+  /// <i class='cupertino-icons md-36'>battery_100</i> &#x2014; cupertino icon for a charging battery.
   /// This is the same icon as [battery_100], [battery_full] and [battery_75_percent] in cupertino_icons 1.0.0+.
   static const IconData battery_charging = IconData(0xf111, fontFamily: iconFont, fontPackage: iconFontPackage);
 
-  /// An empty battery.
+  /// <i class='cupertino-icons md-36'>battery_0</i> &#x2014; cupertino icon for an empty battery.
   /// This is the same icon as [battery_0] in cupertino_icons 1.0.0+.
   static const IconData battery_empty = IconData(0xf112, fontFamily: iconFont, fontPackage: iconFontPackage);
 
-  /// A full battery.
+  /// <i class='cupertino-icons md-36'>battery_100</i> &#x2014; cupertino icon for a full battery.
   /// This is the same icon as [battery_100], [battery_charging] and [battery_75_percent] in cupertino_icons 1.0.0+.
   static const IconData battery_full = IconData(0xf113, fontFamily: iconFont, fontPackage: iconFontPackage);
 
-  /// A 75% charged battery.
+  /// <i class='cupertino-icons md-36'>battery_100</i> &#x2014; cupertino icon for a 75% charged battery.
```

Comment:
Should this be battery_75?

Reply:
There is no battery_75 in the new icon font

Reply:
I'll double check for all your comments, but same thing. The new icon font seems to have collapsed certain icons